### ISSUE TITLE
поиск предметов

### ISSUE THREAD

**Issue by Oreolek** [fireton/mp-newlib#5]
Самое эпическое, что можно дать в помощь игроку.

> &gt; найти халат
Халат находится в гардеробе.

Команда не должна работать, если подходящих предметов больше одного ("стены", "пол"), не обязана автоматически вести игрока к предмету, это просто подсказка где можно найти нужное.

**Comment by fireton** [comment]
В смысле, что предмет может находиться в другой комнате, но игра всё равно пишет что-то вроде «Пробирка в шкафу в лаборатории.»?

**Comment by Oreolek** [comment]
Да, чтобы игрок мог как-то разобраться что где лежит даже если он не тащит с собой сотню предметов.
